cmd/kube-proxy/app/server_others.go

```diff
@@ -50,6 +50,28 @@ import (
 	"k8s.io/klog"
 )
 
+// Reduces a list of bindaddresses down to the first ipv4 and first ipv6 address in the list.
+// (Doesn't check that IP addresses are valid)
+// Returns: []{<ipv4 || ipv6>} || []{<ipv4, ipv6>} || []{<ipv6, ipv4>}
+func parseBindAddresses(addresses []string) []string {
+	filteredAddresses := make([]string, 0)
+
+	hasV4 := false
+	hasV6 := false
+
+	for _, addr := range addresses {
```

Comment:
if we are to do that, then we need to log to user that we ignored (and why) something they provided.